build.bat

```diff
@@ -351,12 +360,19 @@ if !BUILD_CPP! == 1 (
 				echo [ERROR]: In order to link statically using the security plugin you need to also provide the OpenSSL home path by using the --openssl-home option.
 				exit /b 1
 			)
-			set rtiddsgen_extra_options=-additionalRtiLibraries nddssecurity -additionalLibraries "libeay32z ssleay32z"
+			set additional_rti_libs=nddssecurity !additional_rti_libs!
```

Comment:
Make sure no spaces are there if !addition_rti_libs! is empty, or the resulting solution will complain.

Reply:
So far I have not had any complaint from the tool about having `" "` or `"nddssecurity "`. Did you?